src/Hero/README.md

```diff
@@ -0,0 +1 @@
+# Hero
```

Comment:
Perhaps provide a usage example and explanation of props?

Reply:
Added some barebones readme stuff